### ISSUE TITLE
Optimisation

### ISSUE THREAD

**Issue by blt950** [Vatsim-Scandinavia/controlcenter#540]
Now that Handover database is merged into local, let's optimise the queries, specifically:
* Training index for open and closed trainings
* Frequent calls for permissions due to inline checks in views - can a common variable be fed to all views instead?

### Dashboard
BEFORE: avg. loading time: 225ms
AFTER: 351ms? lol

### Activities log
BEFORE: avg. loading time: 1.31s
AFTER: 445ms

### Open trainings
BEFORE: avg. loading time: 2.0s
AFTER: 600ms

### Closed trainings
BEFORE: avg. loading time: 10.4s
AFTER: 2s

### Admin logs
BEFORE: avg. loading time: 6.5s
AFTER: 1.18s
